packages/contracts-bedrock/snapshots/storageLayout/L2OptimismMintableERC20Factory.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "bytes": "32",
+    "label": "spacer_0_0_32",
+    "offset": 0,
+    "slot": "0",
+    "type": "bytes32"
+  },
+  {
+    "bytes": "32",
+    "label": "spacer_1_0_32",
+    "offset": 0,
+    "slot": "1",
+    "type": "bytes32"
+  },
+  {
+    "bytes": "32",
+    "label": "deployments",
+    "offset": 0,
+    "slot": "2",
+    "type": "mapping(address => address)"
+  },
+  {
+    "bytes": "1536",
+    "label": "__gap",
+    "offset": 0,
+    "slot": "3",
+    "type": "uint256[48]"
+  }
+]
```

Comment:
This is net new, but needs to be compared to the previous `OptimismMintableERC20Factory.json`:

```
git checkout develop snapshots/storageLayout/OptimismMintableERC20Factory.json
diff snapshots/storageLayout/OptimismMintableERC20Factory.json snapshots/storageLayout/L2OptimismMintableERC20Factory.json
```


We can see that `_initialized` and `bridge` are removed, and `deployments` is unaffected.
![2024-10-30 at 09 51@2x](https://github.com/user-attachments/assets/44d76237-d75e-4442-80e5-0657e7c64ea4)